TreeView: Add disabledNodeSelectionMode Desc. (#8476)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxTreeView/1 Configuration/disabledNodeSelectionMode.md

---
---
##### shortDescription

> Configures the selection behavior of disabled nodes.

Reviewer: I would mention something like "batch selection". Right now, it may seem like I can maybe select a single disabled node directly. 

```suggestion
Configures disabled node behavior during batch selection operations.
```

---

> This property configures selection behavior for UI selection and methods such as [selectAll()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#selectAll) and [selectItem()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#selectItemkey). The following values are available:

Reviewer: I think we should add details to the first item in the list. We already gave a generic introduction in the short description. Now the exact methods and UI operations can be mentioned in the list item. We can talk in specific terms, instead of referencing a generic "behavior" again. 

```suggestion
The following values are available:
```

> - *"recursiveAndAll"*: Recursive selection and Select All operations can select/unselect disabled nodes.

Reviewer: ```suggestion
- *"recursiveAndAll"*: Recursive selection and Select All operations affect disabled nodes. Selection operations can be initiated by an end user or by the following methods: [selectAll()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#selectAll) and [selectItem()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#selectItemkey). 
```

> - *"recursiveAndAll"*: Recursive selection and Select All operations affect disabled nodes. Selection operations can be initiated by an end user or by the following methods: [selectAll()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#selectAll)/[unselectAll()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#unselectAll) and [selectItem()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#selectItemkey)/[unselectItem()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#unselectItemkey). 

Reviewer 2: ```suggestion
- *"recursiveAndAll"*: Recursive selection and Select All operations affect disabled nodes. Users can initiate selection operations or call the following methods: [selectAll()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#selectAll)/[unselectAll()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#unselectAll), and [selectItem()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#selectItemkey)/[unselectItem()](/Documentation/ApiReference/UI_Components/dxTreeView/Methods/#unselectItemkey). 
```
